stdlib/array.gr

```diff
@@ -583,6 +639,9 @@ provide let fromList = list => {
  * @param search: The value to compare
  * @param array: The array to inspect
  * @returns `true` if the value exists in the array or `false` otherwise
+ * 
+ * @example
+ * Array.contains(9, [> 5, 6, 7, 8, 9]) // true
```

Comment:
Same As Above:
```suggestion
 * Array.contains(9, [> 5, 6, 7, 8, 9]) == true
```

Reply:
I would prefer the in line true.  It starts to get confusing when you are comparing a bool vs. bool with == and not providing the answer with another in line comment that true == true is true.  The as submitted with the in line is crystal clear and immediately you can see that the fn is returning a bool.